silverback/cluster/client.py

```diff
@@ -54,7 +55,7 @@ def render_error(error: dict):
             else:
                 message = response.text
 
-        raise RuntimeError(message)
+        raise click.UsageError(message)
```

Comment:
What I'd like to do that I think might work pretty well is to have a subclass of the base `SilverbackException` class called `ClientError` which also subclasses `click.UsageError` and raise that here.